<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1222</width>
    <height>770</height>
   </rect>
  </property>
  <property name="cursor">
   <cursorShape>ArrowCursor</cursorShape>
  </property>
  <property name="windowTitle">
   <string>道听图说</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="windowWidget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1221</width>
     <height>770</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <family>楷体</family>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>ArrowCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image:url(:/images/newjiemian.png)</string>
   </property>
   <widget class="MapTilesFrame" name="mapFrame">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>180</y>
      <width>750</width>
      <height>500</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>750</width>
      <height>500</height>
     </size>
    </property>
    <property name="sizeIncrement">
     <size>
      <width>750</width>
      <height>500</height>
     </size>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/images/background.png);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="RouteWidget" name="routeWidget" native="true">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>750</width>
       <height>500</height>
      </rect>
     </property>
     <widget class="QLabel" name="locatingLabel">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>190</y>
        <width>161</width>
        <height>61</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>宋体</family>
        <pointsize>14</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">/*border-image: none;*/</string>
      </property>
      <property name="text">
       <string>正在定位 . . .</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QToolButton" name="searchToolButton">
    <property name="geometry">
     <rect>
      <x>960</x>
      <y>160</y>
      <width>70</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Adobe 楷体 Std R</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/images/button.png);
</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="images.qrc">
      <normaloff>:/images/search.png</normaloff>:/images/search.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>55</width>
      <height>55</height>
     </size>
    </property>
   </widget>
   <widget class="QToolButton" name="enlargeToolButton">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>110</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image:none;
background-color: rgba(255, 255, 255, 0);

</string>
    </property>
    <property name="icon">
     <iconset resource="images.qrc">
      <normaloff>:/images/jia.png</normaloff>:/images/jia.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>30</width>
      <height>30</height>
     </size>
    </property>
   </widget>
   <widget class="QToolButton" name="shrinkToolButton">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>110</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image:none;
background-color: rgba(255, 255, 255, 0);</string>
    </property>
    <property name="icon">
     <iconset resource="images.qrc">
      <normaloff>:/images/jian.png</normaloff>:/images/jian.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>30</width>
      <height>30</height>
     </size>
    </property>
   </widget>
   <widget class="QToolButton" name="aboutDlgToolButton">
    <property name="geometry">
     <rect>
      <x>958</x>
      <y>110</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image:none;
background-color: rgba(255, 255, 255, 0);</string>
    </property>
    <property name="icon">
     <iconset resource="images.qrc">
      <normaloff>:/images/wen.png</normaloff>:/images/wen.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>30</width>
      <height>30</height>
     </size>
    </property>
   </widget>
   <widget class="QToolButton" name="xunluToolButton">
    <property name="geometry">
     <rect>
      <x>1030</x>
      <y>160</y>
      <width>70</width>
      <height>50</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/images/button.png);
</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="images.qrc">
      <normaloff>:/images/xunlu.png</normaloff>:/images/xunlu.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>33</width>
      <height>33</height>
     </size>
    </property>
   </widget>
   <widget class="QToolButton" name="locateToolButton">
    <property name="geometry">
     <rect>
      <x>890</x>
      <y>160</y>
      <width>70</width>
      <height>50</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/images/button.png);
</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="images.qrc">
      <normaloff>:/images/locate.png</normaloff>:/images/locate.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>55</width>
      <height>55</height>
     </size>
    </property>
   </widget>
   <widget class="QLabel" name="wozaiLabel">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>215</y>
      <width>70</width>
      <height>70</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image:none;
background-color: rgba(255, 255, 255, 0);
border-image: url(:/images/wozai.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="locationLabel">
    <property name="geometry">
     <rect>
      <x>930</x>
      <y>230</y>
      <width>170</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="sousuoLineEdit">
    <property name="geometry">
     <rect>
      <x>875</x>
      <y>300</y>
      <width>170</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="placeholderText">
     <string>输入搜索地名</string>
    </property>
   </widget>
   <widget class="QToolButton" name="sousuoToolButton">
    <property name="geometry">
     <rect>
      <x>1045</x>
      <y>295</y>
      <width>70</width>
      <height>50</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/images/button.png);
</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="images.qrc">
      <normaloff>:/images/sousuo.png</normaloff>:/images/sousuo.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>35</width>
      <height>35</height>
     </size>
    </property>
   </widget>
   <widget class="QToolButton" name="volumeToolButton">
    <property name="geometry">
     <rect>
      <x>877</x>
      <y>110</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image:none;
background-color: rgba(255, 255, 255, 0);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="images.qrc">
      <normaloff>:/images/shengyin.png</normaloff>:/images/shengyin.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>50</width>
      <height>50</height>
     </size>
    </property>
   </widget>
   <widget class="QToolButton" name="setupDlgToolButton">
    <property name="geometry">
     <rect>
      <x>922</x>
      <y>113</y>
      <width>32</width>
      <height>32</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image:none;
background-color: rgba(255, 255, 255, 0);
</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="images.qrc">
      <normaloff>:/images/function.ico</normaloff>:/images/function.ico</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>40</width>
      <height>40</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="exitPushButton">
    <property name="geometry">
     <rect>
      <x>1050</x>
      <y>110</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image:none;
background-color: rgba(255, 255, 255, 0);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="images.qrc">
      <normaloff>:/images/quit.png</normaloff>:/images/quit.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>30</width>
      <height>30</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>1017</x>
      <y>110</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: none;
background-color: rgba(255, 255, 255, 0);
</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="images.qrc">
      <normaloff>:/images/zuixiaohua.png</normaloff>:/images/zuixiaohua.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>30</width>
      <height>30</height>
     </size>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>590</y>
      <width>100</width>
      <height>100</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/images/frame/kuang4.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>590</y>
      <width>100</width>
      <height>100</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/images/frame/kuang3.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>170</y>
      <width>100</width>
      <height>100</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/images/frame/kuang2.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>170</y>
      <width>100</width>
      <height>100</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/images/frame/kuang1.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QWidget" name="xunluWidget" native="true">
    <property name="geometry">
     <rect>
      <x>880</x>
      <y>210</y>
      <width>220</width>
      <height>480</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image:none;
background-color: rgba(255, 255, 255, 0);</string>
    </property>
    <widget class="QWidget" name="zizhuxunluWidget" native="true">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>210</y>
       <width>220</width>
       <height>180</height>
      </rect>
     </property>
     <widget class="QToolButton" name="zhongdianToolButton">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>60</y>
        <width>81</width>
        <height>51</height>
       </rect>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url(:/images/button.png);</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="images.qrc">
        <normaloff>:/images/zhognidan.png</normaloff>:/images/zhognidan.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>60</width>
        <height>60</height>
       </size>
      </property>
     </widget>
     <widget class="QToolButton" name="qidianToolButton">
      <property name="geometry">
       <rect>
        <x>1</x>
        <y>5</y>
        <width>81</width>
        <height>51</height>
       </rect>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url(:/images/button.png);</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="images.qrc">
        <normaloff>:/images/qidian.png</normaloff>:/images/qidian.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>60</width>
        <height>60</height>
       </size>
      </property>
     </widget>
     <widget class="QToolButton" name="quedingToolButton">
      <property name="geometry">
       <rect>
        <x>78</x>
        <y>115</y>
        <width>80</width>
        <height>50</height>
       </rect>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url(:/images/button.png);
</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="images.qrc">
        <normaloff>:/images/queding.png</normaloff>:/images/queding.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>70</width>
        <height>70</height>
       </size>
      </property>
     </widget>
     <widget class="QLineEdit" name="zhongdianLineEdit">
      <property name="geometry">
       <rect>
        <x>79</x>
        <y>64</y>
        <width>141</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url(:/images/newjiemian.png);</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="placeholderText">
       <string>输入终点</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="qidianLineEdit">
      <property name="geometry">
       <rect>
        <x>79</x>
        <y>10</y>
        <width>141</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url(:/images/newjiemian.png);</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="placeholderText">
       <string>输入起点</string>
      </property>
     </widget>
    </widget>
    <widget class="QToolButton" name="recommendRouteToolButton">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>40</y>
       <width>110</width>
       <height>50</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: url(:/images/button.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="images.qrc">
       <normaloff>:/images/recommend.png</normaloff>:/images/recommend.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>70</width>
       <height>70</height>
      </size>
     </property>
    </widget>
    <widget class="QToolButton" name="beautifulRouteToolButton">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>95</y>
       <width>110</width>
       <height>50</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: url(:/images/button.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="images.qrc">
       <normaloff>:/images/beautiful.png</normaloff>:/images/beautiful.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>70</width>
       <height>70</height>
      </size>
     </property>
    </widget>
    <widget class="QToolButton" name="zizhuxunluToolButton">
     <property name="geometry">
      <rect>
       <x>59</x>
       <y>150</y>
       <width>110</width>
       <height>50</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: url(:/images/button.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="images.qrc">
       <normaloff>:/images/zizhuxunlu.png</normaloff>:/images/zizhuxunlu.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>70</width>
       <height>70</height>
      </size>
     </property>
    </widget>
   </widget>
   <widget class="QToolButton" name="quxiaoButton">
    <property name="geometry">
     <rect>
      <x>943</x>
      <y>620</y>
      <width>111</width>
      <height>51</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/images/button.png);
</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="images.qrc">
      <normaloff>:/images/quxiaoweizhixianshi.png</normaloff>:/images/quxiaoweizhixianshi.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>70</width>
      <height>80</height>
     </size>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>MapTilesFrame</class>
   <extends>QFrame</extends>
   <header>maptilesframe.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>RouteWidget</class>
   <extends>QWidget</extends>
   <header location="global">routewidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections/>
</ui>
